css/styles.css

```diff
@@ -1,6 +1,11 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&display=swap');
+
+body {
+  background-color: lightgray;
+}
```

Comment:
- [ ] There is a grey colored margin in the body of your page which I think was introduced by the default margin given by the body tag. You can remove the default margin by adding `margin: 0;` to the body and see if it fixes the issue 😄

See screenshot below:
<img width="231" alt="Loom_djOULCC1Zn" src="https://user-images.githubusercontent.com/81468675/148488653-c15f8211-4868-49f6-b1d9-bd18314be150.png">